.eslintrc.js

```diff
@@ -76,6 +76,22 @@ module.exports = {
         "internal-rules/multiline-comment-style": "error"
     },
     overrides: [
+        {
+            files: "**/no-useless-escape.md/*.js",
+            rules: {
+                "no-unused-expressions": "off"
+            }
+        },
+        {
+            files: "**/ast-utils.md/*.js",
+            rules: {
+                "no-floating-decimal": "off",
+                "no-octal": "off",
+                "no-unused-expressions": "off",
+                "no-multi-spaces": "off",
+                semi: "off"
+            }
+        },
```

Comment:
It looks like linting jsdoc examples will introduce more problems than benefits if we'll have to override configuration often.

Reply:
While some of the rules could be loosened across all examples, especially in ESLint where you often actually want some bad practices showing up in examples (e.g., the `no-octal` rule there)--as well as the bad practices shown in Markdown example blocks, if you wanted to lint those files' examples, yes, it would involve more frequent tweaking like this.

In my own projects, I have tended to consider the cost worthwhile in ensuring that the examples not only dogfood styles, but work as expected, but I admit it comes at something of a cost (esp. since we have no mechanism similar to the Markdown plugin whereby one can add disable directives just above the jsdoc blocks instead of within them (which would be ugly) or within an external file (which gets less maintainable being out-of-band)).

I can disable the `check-examples` rule then if desired.

Reply:
Yeah, I don’t think we are going to get a ton of benefit from linting examples, so let’s keep it off for now. We can always revisit later.